http/src/request/scheduled_event/create_guild_scheduled_event/stage_instance.rs

```diff
@@ -0,0 +1,76 @@
+use super::{CreateGuildScheduledEvent, CreateGuildScheduledEventFields};
+use crate::{
+    error::Error,
+    request::{AuditLogReason, AuditLogReasonError, Request, TryIntoRequest},
+    response::ResponseFuture,
+};
+use twilight_model::{
+    datetime::Timestamp,
+    id::{marker::ChannelMarker, Id},
+    scheduled_event::{EntityType, GuildScheduledEvent},
+};
+
+/// Create a stage instance scheduled event in a guild.
+#[must_use = "requests must be configured and executed"]
+pub struct CreateGuildStageInstanceScheduledEvent<'a>(CreateGuildScheduledEvent<'a>);
+
+impl<'a> CreateGuildStageInstanceScheduledEvent<'a> {
+    pub(crate) const fn new(
+        inner: CreateGuildScheduledEvent<'a>,
+        channel_id: Id<ChannelMarker>,
+        name: &'a str,
+        scheduled_start_time: &'a Timestamp,
+    ) -> Self {
+        Self(CreateGuildScheduledEvent {
+            fields: CreateGuildScheduledEventFields {
+                channel_id: Some(channel_id),
+                entity_type: Some(EntityType::StageInstance),
+                name: Some(name),
+                scheduled_start_time: Some(scheduled_start_time),
+                ..inner.fields
+            },
+            ..inner
+        })
+    }
+
+    /// Set the description of the event.
+    ///
+    /// Must be between 1 and 1000 characters in length.
+    pub const fn description(mut self, description: &'a str) -> Self {
+        self.0.fields.description = Some(description);
+
+        self
+    }
+
+    /// Set the scheduled end time of the event.
+    ///
+    /// This is not a required field for stage instance events.
```

Comment:
Would users expect it to be required? Feels a bit weird to point out

Reply:
more docs is not always bad.